app/code/Magento/Eav/Model/Entity/Attribute/OptionManagement.php

```diff
@@ -209,12 +220,12 @@ public function getItems($entityType, $attributeCode)
         $attribute = $this->attributeRepository->get($entityType, $attributeCode);
 
         try {
-            $options = $attribute->getOptions();
+            $attribute->setOptions($attribute->getOptions());
```

Comment:
Hello @engcom-Golf ,
thank you for your support

Could you explain why you have used this "workaround" before calling execute of service class?

Reply:
Hi @danielrussob,
Firstly it was added for caching options in attribute object, however, it can work without preload attributes before populate. Updated solution.
Thank you for your assistance.

Reply:
Hello @engcom-Golf , 
thank you for your response, I have this question:

why you have used $attribute->setOptions($attribute->getOptions()) <-- getter in setter on the same "data" ?

Reply:
As I said earlier, for caching options data in attributes object and further multiple usages, as getter does not have caching logic in its implementation.